### PR TITLE
Document that UC1 (the visualization) is facilitated

### PR DESCRIPTION
**Pull request by dmcguire81**
@mustaphatidooyussif I was able to use your cleaned data to support the visualization queries I built against my own normalization. Results were identical, except for the following caveats:
1. Your upper-casing of the `DBA Name`
2. Your `License #` field (erroneously) has a floating point type (should be integer)

@josecols I wasn't able to use you Tableau file for this, because the data wasn't exported (i.e.: it was a `*.twb` and not a `*.twbx`), so I used the one I made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +132,221 @@ sqlite> .read Food_Locations.sql
...
+INNER JOIN Cleaned_Food_Licensees
+ON Cleaned_Food_Licensee_Inspections."License #" = Cleaned_Food_Licensees."License #"
+INNER JOIN Cleaned_Food_Locations
+ON Cleaned_Food_Licensees."Address" = Cleaned_Food_Locations."Address"
+    AND Cleaned_Food_Licensees."City" = Cleaned_Food_Locations."City"
+    AND Cleaned_Food_Licensees."State" = Cleaned_Food_Locations."State"
+    AND Cleaned_Food_Licensees."Zip" = Cleaned_Food_Locations."Zip"
+WHERE Cleaned_Food_Locations."Latitude" BETWEEN 41.9754 AND 42.0034
+    AND Cleaned_Food_Locations."Longitude" BETWEEN -87.8173 AND -87.7284
+    AND strftime('%Y-%m', Cleaned_Food_Licensee_Inspections."Inspection Date") = '2010-08'
```
Comment (josecols) [inline README.md:161]: I am curious about these filters. Are they arbitrary values, or did you select them to make the difference in food location results more evident?

**Review by josecols — COMMENTED** [review]
@dmcguire81 I apologize for the Tableau file; I didn't know there was a format that included the data alongside the visualization. Given that you already posted the query results, should we publish your Tableau visualization instead?

**Comment by dmcguire81** [comment]
> @dmcguire81 I apologize for the Tableau file; I didn't know there was a format that included the data alongside the visualization. Given that you already posted the query results, should we publish your Tableau visualization instead?

I was careful to avoid giving any hint that there is a difference (like the casing), but my visualization was actually built off of files that I made earlier (with only two datasets, not five), because I didn't anticipate needing to wrap it up. Unfortunately, I'm not going to have time to retrofit the visualization to use the cleaned data, as I tried to get an ODBC connector working with SQLite and failed. If either of you is motivated, the simplest way to reproduce what I did would be to export the `Cleaned_` tables as CSV and pull those into Tableau, and it should be able to infer the relationships based on the shared column names. Otherwise, I say we omit a link to the visualization, since the screenshots should be sufficient.

File: README.md (hunk 1 above)
Comment (dmcguire81) [reply README.md:161]: PS: the selected item in each visualization corresponds to the last item in the table results.

**Review by josecols — APPROVED** [review]
I managed to publish the two visualizations using Tableau Public. I also failed to set the ODBC driver because of an issue with Apple Silicon Mac and the published binary for the driver.

For the cleaned visualization, I used the three cleaned tables you added to the Food_Inspections.sqlite file you uploaded yesterday. While the screenshots in this PR focused on the difference in the number of food establishments with coordinates, I concentrated on displaying the Violations filter. I believe these two are the primary data quality improvements we have accomplished.

These are the links for the two dashboards (the .twbx files are in the OneDrive folder):

- https://public.tableau.com/app/profile/jose.cols/viz/Visualization_16595789943680/Dashboard
- https://public.tableau.com/app/profile/jose.cols/viz/CleanedVisualization/Dashboard

**Pull request merged**